File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-02/plagiarized/L5/01/L5.kt
package IR

import java.util.*

object L5 {
    fun hitLuas(jari2: Double): Double {
        return jari2 * jari2 * 3.14159
    }

    fun total(luas: Double, panjang: Double): Double {
        return luas * panjang
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val input = Scanner(System.`in`)
        val luas: Double
        val total: Double
        print("Enter the radius and length of a cylinder: ")
        val jari2 = input.nextDouble()
        val panjang = input.nextDouble()
        luas = hitLuas(jari2)
        total = total(luas, panjang)

        print("The area is $luas , ")
        println("The volume of the cylinder is $total")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L5/01/L5.py
import math

def hit_luas(jari2):
    return jari2 * jari2 * math.pi

def total(luas, panjang):
    return luas * panjang

if __name__ == "__main__":
    jari2, panjang = map(float, input("Enter the radius and length of a cylinder: ").split())
    luas = hit_luas(jari2)
    total_volume = total(luas, panjang)
    print(f"The area is {luas} , The volume of the cylinder is {total_volume}")

